stable/mongodb/templates/deployment-standalone.yaml

```diff
@@ -118,6 +120,9 @@ spec:
         resources:
 {{ toYaml .Values.resources | indent 10 }}
       volumes:
+      - name: custom-init-scripts
```

Comment:
Shouldn't this be optional? Only if: `(.Files.Glob "files/docker-entrypoint-initdb.d/*[sh|js]")`

Reply:
yes